### PR TITLE
ansible teardown playbook

### PR DESCRIPTION
**Pull request by lehoanganh**

### CODE REVIEW

**Comment by rabbah** [comment]
How about a redo teardown target ;)?

File: ansible/teardown.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+# This playbook cleans all docker containers
+
+- hosts: all:!ansible
+  tasks:
+    - name: wait for user confirmation
+      pause: 
+        prompt: > 
+                About to clean ALL docker containers.
```
Comment (rabbah) [inline ansible/teardown.yml]: how do you feel about removing the prompt?

**Comment by rabbah** [comment]
offering `redo` patch for `redo teardown`:

```
diff --git a/tools/build/redo b/tools/build/redo
index dc7e344..84dc9fa 100755
--- a/tools/build/redo
+++ b/tools/build/redo
@@ -231,6 +231,10 @@ Components = [
                   modes = 'clean',
                   gradle = True),
 
+    makeComponent('teardown',
+                  'teardown all deployed containers',
+                  yaml = 'teardown.yml'),
+
```

File: ansible/teardown.yml (hunk 1 above)
Comment (lehoanganh) [inline ansible/teardown.yml]: can do it, no prompt is good

**Comment by lehoanganh** [comment]
can you put your commit on top of mine?

**Comment by rabbah** [comment]
sure i'll squash into your commit.

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/115663229dfe87312ad1e56609ca6c6eeef280ed with `redo teardown`.

**Pull request closed without merging**